### ISSUE TITLE
Bot can't change vanity

### ISSUE THREAD

**Issue by Permanently** [CapingTeam/Vanity-Sniper#1]
Upon looking at logging for vanity requests, this message is returned:

```{ message: 'Bots cannot use this endpoint', code: 20001 }```

**Comment by 002-sans** [comment]
Is your bot on a level 3 server and with the necessary permissions?

**Comment by Permanently** [comment]
Yes to both of those questions. Not sure if the bot can change the vanity any more; from the message that's my understanding, but I can't find any developer document to back that up.
